README.md

```diff
@@ -92,6 +92,15 @@ In the provision-section of the config.tfvars config file
 - **`dashboard_username`**: username to access your kubernetes dashboard
 - **`dashboard_password`**: password for your kubernetes dashboard
 
+*Pachyderm + Minio (optional)*
+- **`TF_VAR_pachyderm_release_name`**: a release name for the Pachyderm service
+- **`TF_VAR_pachyderm_etcd_pvc_size`**: storage dedicated for etcd (In GB)
+- **`TF_VAR_minio_release_name`**: release name for the Minio service
+- **`TF_VAR_minio_pvc_size`**: storage dedicated for the Minio service (In GB)
+- **`TF_VAR_minio_accesskey`**: access key for the S3 endpoint
+- **`TF_VAR_minio_secretkey`**: secret key for the S3 endpoint
+- **`TF_VAR_minio_replicas`**: number of replicas of the Minio service
+
```

Comment:
You should probably remove the TF_VAR_ prefix since it is not used in this version, you can include [skip ci] in the comment of the commit, then you skip travis